projects/configs/tag-operations/base.yaml

```diff
@@ -0,0 +1,660 @@
+- name: "Transmit Coil Name"
+  group: 0x0018
+  element: 0x1251
+  op: "keep"
+#https://dicom.innolitics.com/ciods/mr-image/mr-image/00181251
+#https://dicom.innolitics.com/ciods/mr-spectroscopy/mr-spectroscopy-multi-frame-functional-groups/52009229/00189049/00181251
+#https://dicom.innolitics.com/ciods/enhanced-mr-image/enhanced-mr-image-multi-frame-functional-groups/52009230/00189049/00181251
+#https://dicom.innolitics.com/ciods/enhanced-mr-color-image/enhanced-mr-color-image-multi-frame-functional-groups/52009229/00189049/00181251
+- name: "Flip Angle"
+  group: 0x0018
+  element: 0x1314
+  op: "keep"
+#https://dicom.innolitics.com/ciods/mr-image/mr-image/00181314
+#https://dicom.innolitics.com/ciods/enhanced-mr-image/enhanced-mr-image-multi-frame-functional-groups/52009230/00189112/00181314
+#https://dicom.innolitics.com/ciods/enhanced-mr-color-image/enhanced-mr-color-image-multi-frame-functional-groups/52009229/00189112/00181314
+- name: "Acquisition Device Processing Description"
+  group: 0x0018
+  element: 0x1400
+  op: "keep"
+#https://dicom.innolitics.com/ciods/x-ray-radiofluoroscopic-image/x-ray-image/00181400
+#https://dicom.innolitics.com/ciods/x-ray-angiographic-image/x-ray-image/00181400
+#https://dicom.innolitics.com/ciods/digital-x-ray-image/dx-image/00181400
+#https://dicom.innolitics.com/ciods/digital-mammography-x-ray-image/dx-image/00181400
+#https://dicom.innolitics.com/ciods/digital-intra-oral-x-ray-image/dx-image/00181400
+#https://dicom.innolitics.com/ciods/x-ray-3d-craniofacial-image/x-ray-3d-craniofacial-image-contributing-sources/00189506/00181400
+#https://dicom.innolitics.com/ciods/x-ray-3d-angiographic-image/x-ray-3d-angiographic-image-contributing-sources/00189506/00181400
+#https://dicom.innolitics.com/ciods/breast-projection-x-ray-image/breast-projection-x-ray-image-multi-frame-functional-groups/52009229/00189412/00181400
+- name: "Patient Position"
+  group: 0x0018
+  element: 0x5100
+  op: "replace"
+#https://dicom.innolitics.com/ciods/ultrasound-image/general-series/00185100
+#https://dicom.innolitics.com/ciods/mr-image/general-series/00185100
+#https://dicom.innolitics.com/ciods/ct-image/general-series/00185100
+#NOTE: Seach term: "00185100 Image General Series"
+- name: "MR Acquisition Frequency Encoding Steps"
+  group: 0x0018
+  element: 0x9058
+  op: "keep"
+#https://dicom.innolitics.com/ciods/enhanced-mr-image/enhanced-mr-image-multi-frame-functional-groups/52009230/00189125/00189058
+#https://dicom.innolitics.com/ciods/enhanced-mr-color-image/enhanced-mr-color-image-multi-frame-functional-groups/52009229/00189125/00189058
+- name: "MR Acquisition Phase Encoding Steps in-plane"
+  group: 0x0018
+  element: 0x9231
+  op: "keep"
+#https://dicom.innolitics.com/ciods/enhanced-mr-image/enhanced-mr-image-multi-frame-functional-groups/52009230/00189125/00189231
+#https://dicom.innolitics.com/ciods/enhanced-mr-color-image/enhanced-mr-color-image-multi-frame-functional-groups/52009230/00189125/00189231
```

Comment:
I think should be in MRI.yaml

Reply:
[acc3b3c](https://github.com/SAFEHR-data/PIXL/pull/498/commits/acc3b3cebbbbb8987bc01783ccbc89fee30d51bf)